cmake/common.cmake

```diff
@@ -704,3 +705,31 @@ function(add_picobench name)
 
   set_property(TEST ${name} PROPERTY LABELS benchmark)
 endfunction()
+
+if("sgx" IN_LIST COMPILE_TARGETS)
```

Comment:
It'd be nice to have this in a new `cmake/ravl.cmake` file to be consistent with how we do it for other third-party dependencies.